.github/workflows/builds.yml

```diff
@@ -1961,7 +1961,7 @@ jobs:
     needs: [ prepare-deps, prepare-cbindgen ]
     strategy:
       matrix:
-        dpdk_version: [ 22.11.1, 21.11.3, 20.11.7, 19.11.14 ]
+        dpdk_version: [ "main", 22.11.3, 21.11.5, 20.11.9, 19.11.14 ]
```

Comment:
I think we should drop main for now as its a moving target.  Could be moved to a job that is allowed to fail though.

Reply:
Agree

Reply:
In the end, I decided to omit the main because it was throwing me a warning when I was using an obsolete function and at the same time it was giving me a warning when I was using the new function because it is not part of the stable ABI. 
I guess this was not intended from the DPDK side but it would mean that in either case (with the obsolete or the new function), the job would fail, or with CFLAGS warnings disabled, we would never notice such warnings. I'll try to rethink this.